firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -210,6 +212,8 @@ private HttpsURLConnection openHttpsUrlConnection(
     httpsURLConnection.addRequestProperty(X_ANDROID_PACKAGE_HEADER_KEY, context.getPackageName());
     httpsURLConnection.addRequestProperty(
         X_ANDROID_CERT_HEADER_KEY, getFingerprintHashForPackage(context));
+    httpsURLConnection.addRequestProperty(
+        X_CLIENT_VERSION_HEADER_KEY, String.format("android-sdk/%s", BuildConfig.VERSION_NAME));
```

Comment:
See https://developer.android.com/studio/build/gradle-tips#share-custom-fields-and-resource-values-with-your-app-code and Crashlytics usage of this field here: https://github.com/firebase/firebase-android-sdk/blob/f104b6a3f8478e678c18d978977a41776d7082c0/firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsReportDataCapture.java#L129